examples/appSpec/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "word"
+version = "0.1.0"
+authors = ["Connor Turland <connorturland@gmail.com>"]
+
+[dependencies]
+serde = "1.0.79"
+serde_json = "1.0.30"
+serde_derive = "1.0.79"
+test_utils = { git = "https://github.com/holochain/holochain-rust", branch = "develop" }
```

Comment:
I switched these back to git based dependencies, and that meant updating the code to work with holochain-rust latest